unstructured_platform_plugins/schema/utils.py

```diff
@@ -15,12 +17,16 @@ def from_paramaeter(cls, param: Parameter) -> "TypedParameter":
         )
 
 
-def get_types_parameters(fn: Callable) -> list[TypedParameter]:
+def get_typed_parameters(fn: Callable) -> list[TypedParameter]:
     type_hints = get_type_hints(fn)
     parameters = list(inspect.signature(fn).parameters.values())
     typed_params = []
     for p in parameters:
         typed_param = TypedParameter.from_paramaeter(param=p)
-        typed_param.param_type = type_hints[typed_param.name]
+        if isinstance(typed_param.annotation, str):
+            # remove optional wrapper if 'Optional' not part of the
+            typed_param.param_type = type_hints[typed_param.name]
+        else:
+            typed_param.param_type = typed_param.annotation
```

Comment:
So the logic here is not clear, the comment (btw unfinished?) mentions about `Optional`, but there is no check of such string anywhere.. Please expand comment.

Reply:
Forgot about this, I started adding this before creating my own files for type hints. Will remove.